packages/core/src/lib/createSystem.ts

```diff
@@ -62,6 +62,8 @@ function getSudoGraphQLSchema(config: KeystoneConfig) {
   const lists = initialiseLists(transformedConfig);
   const adminMeta = createAdminMeta(transformedConfig, lists);
   return createGraphQLSchema(transformedConfig, lists, adminMeta, true);
+  // TODO: adminMeta not required
+  // return createGraphQLSchema(transformedConfig, lists, null, true);
```

Comment:
I can't think of a reason why a `sudo` context would have an `adminMeta`, but this would be a breaking change if someone was querying that using `getContext` or similar